airflow/models/mappedoperator.py

```diff
@@ -795,7 +795,10 @@ def parse_time_mapped_ti_count(self) -> Optional[int]:
             if not isinstance(value, MAPPABLE_LITERAL_TYPES):
                 # None literal type encountered, so give up
                 return None
-            total += len(value)
+            if total == 0:
+                total = len(value)
+            else:
+                total *= len(value)
```

Comment:
I think we could have `total = 1` at the beginning of the function instead…?

Reply:
I tried that, and that's how I found the `.expand()` bug. If we can assume we will always have at least _something_ with a length, we could, but this felt more defensive.

Reply:
Let’s do this right now and maybe change it when we fix the `expand` bug. That way we can keep main working correctly.